setup.py

```diff
@@ -17,6 +17,7 @@
     'imageio',
     'pillow',
     'appdirs',
+    'pooch',
```

Comment:
Don't we need to pin this in `requirements_test.txt` as well?

Reply:
Added.